modules/hosted-control-planes-support-matrix.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/index.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hosted-control-planes-support-matrix_{context}"]
+= Compatibility and support matrix
+
+Note the following scope of compatibility and support for hosted cluster version from {product-title} 4.14 to {product-title} 4.15, install type Multicluster engine for Kubernetes operator (MCE), and the management cluster install type is installer-provisioned infrastructure (IPI).
+
+.Compatibility and support matrix
+[cols="1,1,1,1,1"]
+|===
+| Scenarios  | Upgrade type | Management cluster platform  | Hosted cluster platform | MCE version
+
+| Uprade the management cluster, hosted cluster, and node pools from 4.14 to 4.15, upgrade MCE from 2.4 to 2.5
```

Comment:
I think this might be somewhat ambiguous; the order of upgrades should be:
1. MGMT 
2. MCE
3. CP
4. NP